orchestration/Tests/Samples/shared-services/archetype-definition.abs-path.json

```diff
@@ -1,6 +1,6 @@
 {
-    "Subscriptions": "file(/Users/kungumarajnachimuthu/git/azure-devops/azure-devops/2.0/Tests/Samples/shared-services/subscriptions/subscriptions.json)",
-    "ToolkitComponents": "file(/Users/kungumarajnachimuthu/git/azure-devops/azure-devops/2.0/Tests/Samples/shared-services/toolkit-components/toolkit-components.json)",
-    "ArchetypeParameters": "file(/Users/kungumarajnachimuthu/git/azure-devops/azure-devops/2.0/Tests/Samples/shared-services/parameters/archetype-parameters.json)",
-    "ArchetypeOrchestration": "file(/Users/kungumarajnachimuthu/git/azure-devops/azure-devops/2.0/Tests/Samples/shared-services/orchestrations/archetype-orchestration.json)"
+    "Subscriptions": "file(/Users/kungumarajnachimuthu/git/vdc/orchestration/Tests/Samples/shared-services/subscriptions/subscriptions.json)",
```

Comment:
do not push full paths, delete this file and test, it will break our full unit test.

Reply:
Deleted this file.